### ISSUE TITLE
Snap distance

### ISSUE THREAD

**Issue by davidermined** [digitalcreations/MaxTo#342]
2.0.0-rc1 Beta
It would be nice if the "Snap distance" could be set per monitor. 
10 pixels is something way different on a 4k monitor, compared to a 1080p-one.
And woe to the 5 or so people, who actually have an 8k monitor already.
And many people with multi-monitor setups have different sizes and resolutions.

A preview of how much distance x pixels on a monitor actually are would be nice, too.
No idea, how that could be cleanly done, though.

**Comment by vegardlarsen** [comment]
You are right; this definitely should be in logical pixels, and not device pixels. I don't think this will make it into 2.0, though, it will have to be afterwards.

We would have to change the wording to explain this as well.

**Comment by davidermined** [comment]
not my point. Nobody knows or thinks in logical pixels or knows their scaling settings by heart.
Or are consistent with their scaling settings. That would be worse.

I want a visual hint that actually gives me an idea of the distance.
A list lines of varying lengths each corresponding to 5, 10, 20, 40 pixels.
Then make the lines clickable to automatically fill in that value.
That would be one way to do this.

The problem is this:
As a user I don't care about logical pixels. That's weird back-end stuff that nobody wants to know about.
As a user I care about how far the window can get to another window, before the effect triggers.
And I for one, can't go like:
"10 pixels.... hmmmm.... so the width of the monitor is 1920, so 10 pixels is about half a percent.
But with 200% scaling, it's really 1%.
Now I can look at the screen and see that one percent of the width here corresponds to
0.5 cm (which of course I can just visualize perfectly for some reason)"
Nobody (okay, very few people) can do that kind of math or intuitively good enough at translating
pixel numbers into distances on the screen, if they're not somehow familiar and work with small pixel distances.

**Comment by vegardlarsen** [comment]
I agree that some sort of visual indicator would be better here; but I still think scaling this on different DPI scaled monitors would be a good idea.
